test/src/test/java/org/corfudb/integration/CorfuReplicationUpgradeIT.java

```diff
@@ -52,9 +52,11 @@ public class CorfuReplicationUpgradeIT extends LogReplicationAbstractIT {
     private static final int TOTAL_SINK_STATUS_ENTRIES_INIT_SNAPSHOT_SYNC =
         1 + NUM_SNAPSHOT_SYNC_UPDATES_ON_SINK_STATUS_TABLE;
 
-    private static final String UPGRADE_PLUGIN_PATH = "src/test/resources/transport/grpcConfigForUpgrade.properties";
+    private static final String UPGRADE_PLUGIN_PATH =
+            "./test/src/test/resources/transport/pluginConfigUpgradeSource.properties";
```

Comment:
I left a comment for one of the upgrade properties file.  The upgrade plugin path will be the single renamed properties file.